srcpkgs/awesome/template

```diff
@@ -11,17 +11,40 @@ makedepends="libxcb-devel pango-devel xcb-util-devel xcb-util-image-devel
  xcb-util-keysyms-devel xcb-util-wm-devel xcb-util-cursor-devel
  startup-notification-devel imlib2-devel lua53-lgi libxdg-basedir-devel
  gdk-pixbuf-devel lua53-devel dbus-devel libxkbcommon-devel xcb-util-xrm-devel"
-depends="dbus-x11 lua53-lgi>=0.7.2 pango"
+depends="dbus-x11 $(vopt_if jit lua51-lgi lua53-lgi)>=0.7.2 pango"
 short_desc="Highly configurable, next gen framework window manager for X"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="http://awesomewm.org"
 distfiles="https://github.com/awesomeWM/awesome/releases/download/v${version}/awesome-${version}.tar.xz"
 checksum=78264d6f012350b371e339127aca485260bc0aa935eff578ba75ce1a00e11753
 
+build_options="jit"
+desc_option_jit="Use LuaJIT instead of lua"
+
+hostmakedepends+=" $(vopt_if jit 'lua51 LuaJIT lua51-lgi' 'lua lua-lgi')"
+makedepends+=" $(vopt_if jit \
+	'lua51-devel LuaJIT-devel lua51-lgi' \
+	'lua-devel lua-lgi')"
+depends+=" $(vopt_if jit lua51 lua)-lgi>=0.7.2"
```

Comment:
All of these should be inside the variable definitions. And you should only use versioned lua packages (`lua51` and `lua53`, not `lua`).